src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -15,8 +15,12 @@ endif (CORECLR_SET_RPATH)
 
 add_definitions(-DPAL_STDCPP_COMPAT)
 
-set(ENABLE_LLDBPLUGIN ${CLR_CMAKE_PLATFORM_UNIX} CACHE BOOL "Enable building the SOS plugin for LLDB.")
-set(REQUIRE_LLDBPLUGIN ${CLR_CMAKE_PLATFORM_LINUX} CACHE BOOL "Require building the SOS plugin for LLDB.")
+if (${CLR_CMAKE_PLATFORM_LINUX} OR ${CLR_CMAKE_PLATFORM_FREEBSD})
+ set(PLATFORM_SUPPORTS_LLDB_PLUGIN true)
+endif()
+
+set(ENABLE_LLDBPLUGIN ${PLATFORM_SUPPORTS_LLDB_PLUGIN} CACHE BOOL "Enable building the SOS plugin for LLDB.")
```

Comment:
This should still be CLR_CMAKE_PLATFORM_UNIX, no?  I presume you just said OSX can't build it.

Reply:
It cannot (and does not) at the moment. The open question are other UNIXes - like AIX or other *BSD